src/accelerate/utils/operations.py

```diff
@@ -171,6 +171,9 @@ def send_to_device(tensor, device, non_blocking=False, skip_keys=None):
             # `torch.Tensor.to("npu")` could not find context when called for the first time (see this [issue](https://gitee.com/ascend/pytorch/issues/I8KECW?from=project-issue)).
             elif device == torch.device("npu"):
                 device = "npu:0"
+        elif is_xpu_available():
+            if isinstance(device, int):
+                device = f"xpu:{device}"
```

Comment:
maybe also add the case when `device == torch.device("xpu")` just like npu ?

Reply:
Added